### ISSUE TITLE
Minimize qTox to tray when tray icon was clicked

### ISSUE THREAD

**Issue by towlie** [qTox/qTox#2662]
It works this way on linux OS' but somehow not on Windows

**Comment by agilob** [comment]
1. qtox is active
2. left click tray
3. qtox hides to tray

How do you want it to behave?

**Comment by towlie** [comment]
Is this question for me? And yes, that's what I said in the first post. Latest master doesn't do this. It sets qTox into foreground. No matter if qTox is already active.

**Comment by agilob** [comment]
> Latest master doesn't do this

It does it for me oO

**Comment by towlie** [comment]
I tested this on windows and ubuntu. And it doesn't behave like you said there.
On ubuntu it behaves like you right click the icon and on windows it tries to set it to foreground even if it already is.

**Comment by tolikzinovyev** [comment]
@towlie, I recently changed the tray icon behavior in #2586. Did it work for you before that? I tested the new code in Ubuntu 15.10 with Unity and it worked the same as with 0ef39aa06af177061ab53c259ee9c71de2876264: left click on the tray icon is equivalent to the right click. I didn't test it in Windows, sorry.

**Comment by towlie** [comment]
@aszinovyev This is about what happens when you left click the icon. I am used to the behaviour I described in the first post and @agilob decribed in the second.
On windows you made it to always foreground it (even if it already is in foreground). On linux it behaves like you would right click.
Also I know that this behaviour was somehow borked before your PR.
Consider this issue as a request how the tray should behave when you left click it.

**Comment by tolikzinovyev** [comment]
@towlie I'm not sure if the behavior is possible to change for Unity DE. I will try to fix the issue for Windows when I have a chance to do so.

**Comment by towlie** [comment]
@aszinovyev Thank you!

**Comment by zetok** [comment]
> On ubuntu it behaves like you right click the icon and on windows it tries to set it to foreground even if it already is.

Sounds like broken *buntu, nothing that qTox could do about it. And on sane DE/WM (i3wm) it works in the desired way.

@towlie is it still reproducible on windows?

**Comment by TheNain38** [comment]
@zetok Yes it is, when I left click, it is always set to foreground, and also, when you right click on the tray icon, it is always written: "Show" even when qTox is already in the foreground
